src/css/base.css

```diff
@@ -15,7 +15,7 @@
   --transition: 250ms cubic-bezier(0.4, 0, 0.2, 1);
 
   --tablet-container-max-width: 704px;
-  --desktop-container-max-width: 1024px;
+  --desktop-container-max-width: 1088px;

```

Comment:
I have checked the design, the max width for the container on desktop is 1024px, please revert

Reply:
Hi Zakir, please check the content width on the current build, it is less than on the design: 
![image](https://github.com/olexandrd/goit-markup-team-project/assets/143592959/ee4440a9-7b2e-46f7-b96f-f6f28d392d70)

Reply:
You are right, approve